pimd/pim_cmd.c

```diff
@@ -305,9 +311,10 @@ static void pim_show_assert_metric_helper(struct vty *vty,
 	pim_inet4_dump("<addr?>", am.ip_address, addr_str, sizeof(addr_str));
 
 	vty_out(vty, "%-16s %-15s %-15s %-15s %-3s %4u %6u %-15s\n",
-		ch->interface->name, inet_ntoa(ifaddr), ch_src_str, ch_grp_str,
-		am.rpt_bit_flag ? "yes" : "no", am.metric_preference,
-		am.route_metric, addr_str);
+		ch->interface->name,
+		inet_ntop(AF_INET, &ifaddr, buf, sizeof(buf)),
```

Comment:
not pI4?

Reply:
I was leaving the ones with a width spec using strings - is that ... bad?

Reply:
the system works with spacing in the format string.  so why not use it?

Reply:
ok - will fix and push...